src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
For readability, I suggest indenting the two lines above this one:
```
If not already done,...
  https:...
  `gcloud ...`
For more information...
```

Additionally, I think it reads better to s/:// on this line.